step-distribution-agent/pom.xml

```diff
@@ -75,7 +59,7 @@
 						<configuration>
 							<rules>
 								<requireFilesSize>
-									<maxsize>15000000</maxsize>
+									<maxsize>20000000</maxsize>
```

Comment:
The size of the distribution should not increase with this PR, why this change?

Reply:
Because I'm constantly getting failing distribution builds on other branches when the dependencies get updated, and the sizes have to be constantly adjusted upwards, which can get pretty frustrating after a while. For instance here: https://github.com/exense/step-enterprise-distribution/compare/master...SED-3511-experimental-branch-without-sikulix.

This was therefore a **proactive** change because the sizes will increase anyway, and this check is really only meant to prevent the distribution from accidentally going completely out of bounds. That's also why I changed it to 20M instead of the bare minimum, as it would only be a matter of time until we reach the next unexpected build failure only related to size checks...